cli.js

```diff
@@ -2,13 +2,18 @@
 var Cabal = require('cabal-core')
 var swarm = require('cabal-core/swarm.js')
 var minimist = require('minimist')
+var fs = require('fs')
+var yaml = require('js-yaml')
+
 var frontend = require('./neat-screen.js')
 
 var args = minimist(process.argv.slice(2))
 
 var homedir = process.env.HOME || process.env.HOMEPATH || process.env.USERPROFILE
```

Comment:
Would it make sense to use https://nodejs.org/api/os.html#os_os_homedir ?

```suggestion
var homedir = os.homedir()
```

Reply:
Oh! I like that! Thanks @joehand !